vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -863,7 +874,7 @@ function Test-AzureExtensions {
     if (Test-Path "C:\Packages\Plugins") {
         $actualExtensions = (Get-ChildItem "C:\Packages\Plugins").Name
         if ($actualExtensions.Length -gt 0) {
-            Write-Log "Azure extensions are not expected. Details: $($actualExtensions | Out-String)"
+            Write-Log "Azure extensions are not expected. Skipping for now. Details: $($actualExtensions | Out-String)"
```

Comment:
The log is confusing. We do not ignore it in check-in and prod. We only ignore it in testing vhds. We may need to update the wiki to change these two lines in testing vhds.

Reply:
Reverted the change to the log.